### ISSUE TITLE
Be able to place duplicate buttons in config file

### ISSUE THREAD

**Issue by abondis** [xournalpp/xournalpp#1683]
**Is your feature request related to a problem? Please describe.**
I like the floating toolbox, but I also like to select colors on the side. Having the same color showed in the sidebar and the floating toolbox would be helpful.

Config file does not seem to permit having the same tool declared in different toolbars

**Describe the solution you'd like**
Make the config file accept/work with duplicated tools declarations

**Describe alternatives you've considered**
I don't see any...

**Additional context**
I tried to add the red color selector in the side toolbar and the floating toolbox. Doing so from the UI is not possible since it just moves the tool from toolbar to toolbar. But it seems doable since by default the view with the floating toolbox lets select red, and red is also on the sidebar.
Using the config file I had no luck either, but if it is already doable don't hesitate to point me towards the doc.

**Comment by jvwowd** [comment]
I was able to do this with colors by directly editing the toolbar.ini file (plaintext file and on Linux it is ~/.xournalpp/toolbar.ini). If you go to View -> Toolbars -> Manage and copy the default Floating Toolbox, then there will be an entry for this copy in the toolbar.ini file which you can edit accordingly.

I was unable to use this approach to have duplicates of other toolbar buttons though.

**Comment by felurx** [comment]
I would also really like to be able to have, for example, the pen, eraser and highlighter both in my toolbar and floating toolbox. I would just like to ask, why is implementing this considered hard? @Technius Just curious.

**Comment by Technius** [comment]
It's not so much hard as it is a lot of work. Need to ensure that each tool widget has consistent states, that there aren't any deadlocks of any sort, etc.

**Comment by bhennion** [comment]
The task will simplify a lot once #5154 is merged.
